src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/Mock/MockStream.cs

```diff
@@ -222,10 +222,16 @@ internal sealed class StreamState
             public long _outboundErrorCode;
             public long _inboundErrorCode;
 
+            private const int InitialBufferSize =
+#if DEBUG
+                10;
+#else
+                4096;
+#endif
```

Comment:
Can we similarly set the max buffer size to something very small?

Reply:
Sure. I made it 64.

Reply:
Making it so small is making the "large data" test take forever to run.

Even at 4096 bytes, it still takes several seconds to run.

I'm going to set this to 4096 and reduce the size in the test to 16MB (from 64MB).